cvmfs/util.cc

```diff
@@ -886,6 +886,35 @@ bool HasPrefix(const string &str, const string &prefix,
 }
 
 
+bool HasSuffix(
```

Comment:
According to my mini benchmark: no significant difference in size or performance (gcc 4.4 and OS X 10.9 clang).  Changed.